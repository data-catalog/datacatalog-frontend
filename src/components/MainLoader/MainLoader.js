import React from 'react';
import styled from 'styled-components';
import Loader from 'react-loader-spinner';
import 'react-loader-spinner/dist/loader/css/react-spinner-loader.css';

const LoaderContainer = styled.div`
  display: flex;
  justify-content: center;
  transform: translateY(70%);
`;

const MainLoader = () => (
  <LoaderContainer>
    <Loader type="Grid" color="#00BFFF" height={200} width={200} />
  </LoaderContainer>
);

export default MainLoader;
