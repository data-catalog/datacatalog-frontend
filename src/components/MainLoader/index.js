import MainLoader from './MainLoader';

export default MainLoader;
