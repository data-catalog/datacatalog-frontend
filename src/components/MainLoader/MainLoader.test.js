import React from 'react';
import { shallow } from 'enzyme';
import MainLoader from './MainLoader';

describe('MainLoader', () => {
  it('renders correctly', () => {
    const tree = shallow(<MainLoader />);
    expect(tree).toMatchSnapshot();
  });
});
