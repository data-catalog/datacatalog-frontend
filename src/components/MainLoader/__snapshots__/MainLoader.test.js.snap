// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainLoader renders correctly 1`] = `
<styled.div>
  <Loader
    className=""
    color="#00BFFF"
    height={200}
    style={Object {}}
    timeout={0}
    type="Grid"
    visible={true}
    width={200}
  />
</styled.div>
`;
